From 3b630045052e1e4d563207ab9e3be8d137c26067 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyich@gmail.com>
Date: Fri, 2 Aug 2024 22:57:35 +0100
Subject: [PATCH] mkvparser/mkvparser.cc: add missing <cstdint> include

Without the change `libwebm` build fails on upcoming `gcc-15` as:

    [  2%] Building CXX object CMakeFiles/mkvparser.dir/mkvparser/mkvparser.cc.o
   mkvparser/mkvparser.cc: In function 'long int mkvparser::UnserializeFloat(IMkvReader*, long long int, long long int, double&)':
    mkvparser/mkvparser.cc:249:7: error: 'uint32_t' does not name a type
      249 |       uint32_t ff;
          |       ^~~~~~~~

Change-Id: I6c9ea3ea761f51812f6507f6c928a0915603eb59
---
 mkvparser/mkvparser.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libwebm-libwebm-1.0.0.31/mkvparser/mkvparser.cc b/libwebm-libwebm-1.0.0.31/mkvparser/mkvparser.cc
index eddbc7eb5..042a0c56c 100644
--- a/libwebm-libwebm-1.0.0.31/mkvparser/mkvparser.cc
+++ b/libwebm-libwebm-1.0.0.31/mkvparser/mkvparser.cc
@@ -16,6 +16,7 @@
 #include <cfloat>
 #include <climits>
 #include <cmath>
+#include <cstdint>
 #include <cstring>
 #include <memory>
 #include <new>